packages/js/src/import.js

```diff
@@ -274,12 +282,34 @@ function initElements() {
  */
 function watchSelect( dropdown ) {
 	var button = dropdown.closest( "form" ).find( "input[type=submit]" );
+	var selectedPlugin, text, textSource;
+
 	dropdown.on( "change", function() {
-		if ( jQuery( this ).find( "option:selected" ).attr( "value" ) === "" ) {
+		selectedPlugin = jQuery( this ).find( "option:selected" ).attr( "value" );
+
+		// Disable the Import button if no button is selected.
+		if ( selectedPlugin === "" ) {
 			button.prop( "disabled", true );
 			return;
 		}
 		button.prop( "disabled", false );
+
+		// Display the relevant text depending on which plugin is selected for import.
+		if ( dropdown === importDropdown ) {
+			text = window.yoastImportData.assets.replacing_texts.select_header.replace( /%s/g, jQuery( this ).find( "option:selected" ).text() );
```

Comment:
Importing `sprintf` from `@wordpress/i18n` to use would I think be more clear here.